packages/@webex/internal-plugin-calendar/src/calendar.decrypt.helper.js

```diff
@@ -36,6 +36,14 @@ const DecryptHelper = {
         )
       : [];
 
+    // Decrypt encryptedScheduleFor properties if meeting object contains SOB
+    const encryptedScheduleFor = Promise.all(
```

Comment:
question: The name here, encryptedScheduleFor. Is it encrypted at this point? I ask because everything else in the function starts with `decrypted`

Reply:
Sorry, it is my typo error, I have changed to to decrypted pre-fix.